eq_translations/survey_schema.py

```diff
@@ -69,6 +70,12 @@ def get_core_pointers(self):
             pointers.extend(key_pointers)
         return pointers
 
+    def get_first_description_pointer(self):
+        try:
+            return [[key_pointer for key_pointer in find_pointers_to(self.schema, 'description')][0]]
+        except IndexError:
+            return []
```

Comment:
This should be named something like get_schema_description_pointer() in order to better identify what's being returned.

The schema description is always found at the pointer '/description'. You should only need to check if the key 'description' is in the schema and return this pointer.

Reply:
I used to call it get_schema_description_pointer but I changed it to "first" , to make it more readable to other people, so basically going round in circles

Reply:
I feel like your original name was more readable. First doesn't explicitly state why that description pointer is unique.